scr/controllers/controllers.js

```diff
@@ -0,0 +1,183 @@
+const controller = {};
+const {connector} = require('../../config/mysql_db')
+const  multer = require('multer')
+const storage = require('../../config/multer');
+const uploader = multer({storage});
+
+
+controller.list = (req, res) => {
+        let sql = 'SELECT * FROM new_product';
+        connector.query(sql, (err, results) => {
+            if(err) throw err;
+            res.render('products.ejs', {title : "Product list", results })
```

Comment:
The static strings should be placed in an another file or at least as grouped as dictionary with proper . Please do some research about the concept of localizable strings nodejs